README.md

```diff
@@ -8,7 +8,8 @@ Ballerina Regex Library [DEPRECATED]
   [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-regex.svg)](https://github.com/ballerina-platform/module-ballerina-regex/commits/main)
   [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/regex.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fregex)
 
-**Deprecation Notice:** This library is deprecated and will no longer be maintained or updated. We recommend that you migrate to [ballerina/lang.regexp](https://lib.ballerina.io/ballerina/lang.regexp/latest) library for continued support and updates.
+**Deprecation Notice:** This library is deprecated and will no longer be maintained or updated. Instead, it is recommended to use the [ballerina/lang.regexp](https://lib.ballerina.io/ballerina/lang.regexp/latest) library for continued support and updates. You can find the new regexp types example [here](https://ballerina.io/by-example/regexp-type), the new regexp operations example [here](https://ballerina.io/by-example/regexp-operations) and the regexp feature guide [here](https://ballerina.io/learn/distinctive-language-features/advanced-general-purpose-language-features/#regular-expressions
```

Comment:
```suggestion
**Deprecation Notice:** This library is deprecated and will no longer be maintained or updated. Instead, it is recommended to use the [`ballerina/lang.regexp`](https://lib.ballerina.io/ballerina/lang.regexp/latest) library for continued support and updates. For more information, see the new [regexp types example](https://ballerina.io/by-example/regexp-type), [regexp operations example](https://ballerina.io/by-example/regexp-operations), and [regexp feature guide](https://ballerina.io/learn/distinctive-language-features/advanced-general-purpose-language-features/#regular-expressions).
```
We should not be adding hyperlinks with the word `here1`. Instead, the best practice is to have the page/BBE title itself as the hyperlink.

Reply:
Fixed in d934dae